src/core/blogs.md

```diff
@@ -0,0 +1,51 @@
+
+## Context
+
+The Rust project maintains two blogs. The “main blog” (blog.rust-lang.org) and a “team blog”. This document provides the guidelines for what it takes to write a post for each of those blogs, as well as how to propose a post and to choose which blog is most appropriate.
+
+## How to select the right blog: audience
+
+So you want to write a Rust blog post, and you’d like to know which blog you should post it on? Ultimately, there are three options:
+
+
+- The main Rust blog
+    - Suitable when your audience is “all Rust users or potential users”
+    - Written from an “official position”, even if signed by an individual
+- The team Rust blog
+    - Suitable when your audience is “all Rust contributors or potential contributors”
+    - Written from an “official position”, even if signed by an individual
+- Your own personal blog
+    - Everything else
+
+There are two key questions to answer in deciding which of these seems right:
+
+- Am I speaking in an “official capacity” or as a “private citizen”?
+- Who is the audience for your post?
+
+In general, if you are speaking as a “private citizen”, then you are probably best off writing on your own personal blog.
+
+If however you are writing in an **official capacity**, then one of the Rust blogs would be good fit. Note that this doesn’t mean you can’t write as an individual. Plenty of the post’s on Rust’s blog are signed by individuals, and in fact that is the preferred option. However, those posts are typically documenting the official position of a team — a good example is Aaron Turon’s classic post on [Rust’s language ergonomics initiative](https://blog.rust-lang.org/2017/03/02/lang-ergonomics.html). Sometimes, the posts are describing an exciting project, but again in a way that represents the project as a whole (e.g., ManishGoregaokar’s report on [Fearless Concurrency in Firefox Quantum](https://blog.rust-lang.org/2017/11/14/Fearless-Concurrency-In-Firefox-Quantum.html)).
+
+To decide between the main blog and the team blog, the question to ask yourself is **who is the audience** for your post. Posts on the main blog should be targeting **all** Rust users or potential users — they tend to be lighter on technical detail, and written without requiring as much context. Posts on the team blog can assume a lot more context and familiarity with Rust.
+
+## Writing for the Main Rust blog
+
+The core team ultimately decides what to post on the main Rust blog.
+
+Post proposals describing exciting developments from within the Rust org are welcome, as well as posts that describe exciting applications of Rust. We do not generally do “promotional cross-posting” with other projects,  however.
+
+If you would like to propose a blog post for the main blog, please reach out to a [core team member](https://www.rust-lang.org/governance/teams/core). It is not suggested to just open PRs against the main Rust blog that add posts without first discussing it with a core team member.
```

Comment:
That's exactly what the release team does tho with the release blog posts (we don't talk these through with T-Core beforehand, except maybe implicitly through Mark's presence but really not)... it seems to me that the release blog posts should be mentioned and it should be noted that the release team handles those since these are by far the most common use of the main blog.

Reply:
Worth noting that this is a pretty recent change, as it used to be me that wrote the posts.

That being said, I agree that it's worth calling out *why* this rule exists, which is why the release team is exempt here: we know that we want a blog post for every release. They're like, pre-authorized or something.

Reply:
@Centril that sounds great -- do you think you could write-up a little text about that? I don't really know how the process works.

Reply:
I would suggest a subsection

Reply:
I'll try to cook something up. :)

Reply:
I wrote a vague description, @Centril, and you can either post the rest here or else as a follow-up PR.

Reply:
Writing it right now :)